frontend/src/features/apartment/ApartmentListPage.tsx

```diff
@@ -59,7 +59,9 @@ const ApartmentListItem = ({
                         <br />
                         {`${address.postal_code} ${address.city}`}
                     </div>
-                    <div className="area">{`${surfaceArea || ""} m² ${rooms || ""} ${apartmentType}`}</div>
+                    <div className="area">{`${surfaceArea || ""}${surfaceArea ? "m²" : ""} ${
+                        rooms || ""
+                    } ${apartmentType}`}</div>
```

Comment:
`${surfaceArea || ""}${surfaceArea ? "m²" : ""}` can be simplified into `${surfaceArea ? surfaceArea+"m²" : ""}`

Reply:
👍 done